apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -328,136 +330,141 @@ function ListBoxInline({ options, layout }) {
   return (
     <>
       {showDetachedToolbarOnly && <ActionsToolbar direction={direction} {...getActionToolbarProps(true)} />}
-      <StyledGrid
-        className="listbox-container"
-        container
-        tabIndex={keyboard.enabled ? -1 : undefined}
-        direction="column"
-        gap={0}
-        containerPadding={containerPadding}
-        style={{ height: '100%', minHeight: `${minHeight}px`, flexFlow: 'column nowrap' }}
-        onKeyDown={handleKeyDown}
-        onMouseEnter={handleOnMouseEnter}
-        onMouseLeave={handleOnMouseLeave}
-        ref={(el) => {
-          containerRef.current = el;
-          containerRectRef(el);
-        }}
-        isGridMode={isGridMode}
-      >
-        {showToolbarWithTitle && (
-          <Grid
-            item
-            container
-            minHeight={headerHeight}
-            flexDirection={isRtl ? 'row-reverse' : 'row'}
-            marginY={1}
-            paddingLeft={`${headerPaddingLeft}px`}
-            paddingRight={`${headerPaddingRight}px`}
-            wrap="nowrap"
-          >
-            {showIcons && (
-              <Grid item container alignItems="center" width={iconsWidth}>
-                {isLocked ? lockIconComp : showSearchIcon && searchIconComp}
-                {isDrillDown && (
-                  <DrillDownIcon
-                    tabIndex={-1}
-                    title={translator.get('Listbox.DrillDown')}
-                    size="large"
-                    style={{ fontSize: '12px' }}
-                  />
-                )}
-              </Grid>
-            )}
+      <ListBoxInputContext.Provider value={liveInputRegionRef}>
```

Comment:
This context provider currently has one purpose only, and that is to bring the screen reader text to this (higher) level in the DOM hierarchy. Maybe there is no other case where we need to do this currently, but you could still write it so it's easy to make it more generic, if we would need to send more properties upwards like this:

Instead of:
```
const liveInputRegionRef = useRef(null);
```

Write:
```
const listBoxContext = useRef(null);
const { liveInputRegionRef, futureProperty } = listBoxContext;   // `futureProperty` is just made up
```